### PR TITLE
chore(plugins): Split first party plugins into seperate repos

### PR DESCRIPTION
**Pull request by sjelin**
Part of https://github.com/angular/protractor/issues/2674

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -290,145 +290,12 @@ exports.addWarning(message, options);
 If you specify any of these properties in your plugin file, they will be
 overwritten.
```
Comment (juliemr) [inline docs/plugins.md]: The docs above this need to be updated as well. they mention build in plugins, and the 'in this document' table of contents is wrong now.

File: docs/plugins.md [hunk 2]
```diff
@@ -290,145 +290,12 @@ exports.addWarning(message, options);
...
-};
-```
-
-Note that this plugin's behavior is undefined on browsers other than Chrome.
-Firefox users have reported flaky results.
-
-Community Plugins
------------------
+First Party Plugins
+-------------------
```
Comment (juliemr) [inline docs/plugins.md]: Why are we killing the "Community Plugins" docs?

File: docs/plugins.md [hunk 3]
```diff
@@ -290,145 +290,12 @@ exports.addWarning(message, options);
...
 
-This list is here for reference and the plugins included are not developed or mantained by protractor's team by any means. If you find any issues with this plugins please report them to the corresponding plugin developer.
+We maintain four plugins:
 
-* [Protractor testability plugin](https://github.com/alfonso-presa/protractor-testability-plugin): this plugins enables synchronous testing with protractor for features that are not developed using the services provided by AngularJS, preventing the need of additional waits coded in the tests. This happens for example if you have WebSockets communication with the server or for web applications built with frameworks different than AngularJS.
+* [Console Plugin (chrome only)](https://www.npmjs.com/package/protractor-console-plugin)
```
Comment (juliemr) [inline docs/plugins.md]: Can we maintain the first-paragraph blurbs of what these plugins do? `Timeline Plugin` means very little without context.

**Comment by juliemr** [comment]
Yay! I'm very excited about this.

We should make sure that we have the new plugin repos running Travis at some point.

File: docs/plugins.md (hunk 2 above)
Comment (sjelin) [inline docs/plugins.md]: It was a mistake.  I'll put it back.

**Comment by sjelin** [comment]
I'll make issues for CI for the plugins.

**Comment by sjelin** [comment]
All comments addressed

**Comment by juliemr** [comment]
Great, LGTM!

**Pull request merged**